draft-ietf-dprive-dnsoquic.md

```diff
@@ -503,14 +508,16 @@ included here.
  
 Servers MAY support session resumption, and MAY do that with or without supporting
 0-RTT, using the mechanisms described in {{Section 4.6.1 of RFC9001}}.
-Servers supporting 0-RTT MUST NOT execute non replayable transactions received in 0-RTT
+Servers supporting 0-RTT MUST NOT take any action in
+response to non replayable transactions received in 0-RTT
 data, and MUST adopt one of the following behaviors:
```

Comment:
I find 'not take any action' confusing because we then prescribe the actions the server must take including sending a response with an EDE code..... I think the intention is that the server not immediately process the DNS transaction and send the processed response prior to the handshake?

```suggestion
Servers supporting 0-RTT MUST NOT immediately process a
replayable transactions received in 0-RTT data, but instead
MUST adopt one of the following behaviours:
```

Reply:
Shouldn't that be "non replayable?"